Core/GraphQL/GraphQLFiles/sell_getMyPaymentIntent.graphql

```diff
@@ -0,0 +1,21 @@
+query getMyPaymentIntent($id: ID!) {
+  getMyPaymentIntent(id: $id) {
+    id
```

Comment:
Is there any `PaymentIntent` Fragment at the moment? If not, it may be worthwhile to create it.

Reply:
I've add fragment for `PaymentIntent`, thank you.